doc/install.md

```diff
@@ -89,13 +89,13 @@ $ sudo apt update
 If you use `cargo` from the repository
 
 ```bash
-$ sudo apt install gcc-aarch64-linux-gnu g++-aarch64-linux-gnu libc6-dev:arm64 libstd-rust-dev:arm64
```

Comment:
Is it needed (here and below)?

Reply:
Depends. If you decide for a debootstrap-only approach it's not needed here. It's definitely needed within the debootstrap chroot. But if you build for multiple targets - some for a different os version and some for the same one as the build host - then it's needed in both places.

Reply:
Or what do you mean with "it"? The whole line or just the g++ package?

Reply:
Actually it's a section of the document I didn't touch. But it's the same typo as in PR #800 so I fixed it here anyway.